consoles/node.html

```diff
@@ -20,7 +20,7 @@ <h1>Node</h1>
 {{ range query "up{job='node'}" | sortByLabel "instance" }}
 <tr>
   <td><a href="node-overview.html?instance={{ .Labels.instance }}">{{ reReplaceAll "(.*?://)([^:/]+?)(:\\d+)?/.*" "$2" .Labels.instance }}</a></td>
-  <td{{ if eq (. | value) 1.0 }}>Yes{{ else }} class="alert-danger">No{{ end }}</td>
+  <td> {{ if eq (. | value) 1.0 }}>Yes{{ else }} class="alert-danger">No{{ end }}</td>
```

Comment:
can you revert this changes. Excepting if you are able to explain what problems you are trying to solve, the code is fine like stated in #10923.

Reply:
ok，thanks！

Reply:
The original is valid.